mathics/eval/arithmetic.py

```diff
@@ -49,6 +49,8 @@
 RealOne = Real(1.0)
 
 
+use_sympy_for_arithmetic = False
```

Comment:
Setting this variable to `True` we use a `eval_Times` and `eval_Plus` implementation based on sympy. Notice that it fails to pass several tests.